.pipelines/ci.yml

```diff
@@ -36,167 +36,125 @@ variables:
+    jobs:
+      - job: Python_Unit_Tests
+        pool:
+          name: 1es-aro-ci-pool
+        variables:
+          HOME: $(Agent.BuildDirectory)
+
+        steps:
+          - template: ./templates/template-checkout.yml
+          - script: |
+              set -xe
+              make test-python
+              [[ -z "$(git status -s)" ]]
+            displayName: 🧪Run Python Unit Tests
+            target: python
+
+      - job: Golang_Unit_Tests
+        pool:
+          name: 1es-aro-ci-pool
+        variables:
+          GOCACHE: /tmp/gocache
+        steps:
+          - template: ./templates/template-checkout.yml
+
+          - script: |
+              set -xe
+              go version
+              go env
+            displayName: Print Go version & env
+            target: golang
+
+          - script: |
+              echo "##vso[task.prependpath]$(go env GOPATH)/bin"
+            displayName: Add GOBIN to path
+            target: golang
+
+          - script: |
+              set -xe
+              # Required for podman 5
+              sudo tdnf install -y gpgme-devel lvm2-devel btrfs-progs-devel golang-1.21.11-1.cm2
+              make generate
+              [[ -z "$(git status -s)" ]]
+            displayName: ⚙️ Run Golang code generate
+            target: golang
+
+          - script: |
+              set -xe
+              make build-all
+              [[ -z "$(git status -s)" ]]
+            displayName: 🕵️ Build Golang code
+            target: golang
+
+          - script: |
+              set -xe
+              make unit-test-go
+            displayName: 🧪 Run Golang unit tests
+            target: golang
+
+          - task: PublishTestResults@2
+            displayName: 📊 Publish tests results
+            inputs:
+              testResultsFiles: $(System.DefaultWorkingDirectory)/**/report.xml
+            condition: succeededOrFailed()
+
+          - script: |
+              make xmlcov
+            displayName: ⚙️ Process Reports
+            condition: succeededOrFailed()
+            target: golang
+
+          - task: PublishCodeCoverageResults@2
+            displayName: 📈 Publish code coverage
+            inputs:
+              codeCoverageTool: Cobertura
+              summaryFileLocation: $(System.DefaultWorkingDirectory)/**/coverage.xml
+              failIfCoverageEmpty: false
+            condition: succeededOrFailed()
+            target: golang
+
+      - job: Lint_Admin_Portal
+        pool:
+          name: 1es-aro-ci-pool
+        steps:
+          - template: ./templates/template-checkout.yml
+          - script: |
+              set -xe
+              make lint-admin-portal
+            displayName: 🧹 Lint Admin Portal
+
+  - stage: Containerized
+    dependsOn: []
+    jobs:
+      - template: templates/template-podman.yml
+        parameters:
+          jobName: 'Build_Test_And_Push_Az_ARO_Extension'
+          makeCommand: 'make ci-azext-aro'
+          repository: 'my-image-repo'
```

Comment:
If I'm reading this correctly `repository` is the same for all images. Let's fix that so that each image build is a unique name that is representative of the artifact (should probably be in sync with the Makefile `LOCAL_*_IMAGE` names).

Reply:
Updated the pipeline to use unique repository names by referencing LOCAL_*_IMAGE variables from the Makefile. This ensures image names are consistent and in sync with the Makefile definitions.